src/plotting.jl

```diff
@@ -79,6 +79,10 @@ function lbllz()
   ylabel(L"\langle Z \rangle")
 end
 
+function lbllpdeg()
+  ylabel("Phase (deg)")
+end
```

Comment:
This whole series of labeling functions strikes me as very opaque. Maybe we should move them to a "Diego" package :).